lib/salus/scanners/osv/gradle_osv.rb

```diff
@@ -76,8 +76,10 @@ def match_vulnerable_dependencies(dependencies)
         if dependency['version'].present?
           version = dependency['version']
           # Cleanup version string to handle case like -
-          # 1.2.1.somestring / 9999.0-empty-to-avoid-conflict-with-test /
-          # 30.3.0-deprecated-use-gradle-api
+          # Dynamic resolved version: dependency.xx.xx:1.1.1 -> 1.5.0
+          # Version strings: 1.2.1.somestring / 9999.0-empty-to-avoid-conflict-with-test /
+          # Version strings: 30.3.0-deprecated-use-gradle-api
+          version = version.split("->")[1].strip if version.include? "->"
```

Comment:
Can we use **last** instead of array_index in the case? Is the following code more readable?

`version.split("->").last.strip`

WDYT?

Here is a test

```
irb(main):001:0> s = "a->b"
=> "a->b"
irb(main):002:0> s.split('->')
=> ["a", "b"]
irb(main):003:0> s.split('->').last
=> "b"
```

Reply:
Yup let me use that convention